debian/control

```diff
@@ -0,0 +1,17 @@
+Source: buildah
+Section: devel
+Priority: optional
+Maintainer: Lokesh Mandvekar <lsm5@fedoraproject.org>
+Build-Depends: debhelper (>=9), rsync, libassuan-dev,
```

Comment:
Are these suffcient?    In our README.md we list: make, golang (Requires version 1.8.1 or higher.), bats, btrfs-progs-devel, device-mapper-devel, gpgme-devel, libassuan-devel, git, bzip2, go-md2man, runc, and skopeo-containers.  I see some of these in the list below, but not all as I would expect.  But I've not done debian packaging in the past so I could be off base.